R/pkg/R/SQLContext.R

```diff
@@ -584,7 +584,7 @@ tableToDF <- function(tableName) {
 #'
 #' @param path The path of files to load
 #' @param source The name of external data source
-#' @param schema The data schema defined in structType
+#' @param schema The data schema defined in structType or a DDL-formatted string.
```

Comment:
but it is `data schema ... in a DDL-formatted string`. I get the point but I am quite sure users would probably less confused (and example is also provided). I actually don't know how I can explain this shorter and cleaner ...